pkg/cli/server/root_test.go

```diff
@@ -2028,6 +2028,141 @@ func TestUpdateLDAPConfig(t *testing.T) {
 	})
 }
 
+func TestClusterConfig(t *testing.T) {
+	baseExamplePath := "../../../examples/scale-out-cluster-cloud/"
+
+	Convey("Should successfully load example configs for cloud", t, func() {
+		for memberIdx := 0; memberIdx < 3; memberIdx++ {
+			cfgFileToLoad := fmt.Sprintf("%s/config-cluster-member%d.json", baseExamplePath, memberIdx)
+			cfg := config.New()
+			err := cli.LoadConfiguration(cfg, cfgFileToLoad)
+			So(err, ShouldBeNil)
+		}
+	})
+
```

Comment:
add a test case for validating the tls configs too to verify that they are correct.

Reply:
Added a test for this.